### ISSUE TITLE
TYPO3 12 compatibility

### ISSUE THREAD

**Issue by zenoussi** [Hauer-Heinrich/hh_video_extender#36]
Hey!
Just wanted to ask if you plan to support TYPO3 v12 and if yes if you got a time plan for that?

Thanks a lot!

**Comment by Teisi** [comment]
yes and no, but as soon as possible - however, other extensions, such as the hh_slider / hh_accordion have priority
